metrics/mobile-app-metrics.md

```diff
@@ -0,0 +1,131 @@
+# Abstract
+
+This document contains the proposal for gathering metrics on system events for running Mobile Apps and making them available to developers.
```

Comment:
Metrics - are usually connected to some changing values: like performance metrics etc.
In this case statistics may be better name.